src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java

```diff
@@ -99,10 +99,28 @@
      */
     private HashSet<String> navigatorProjects = new HashSet<>();
 
+    /**
+     * To store if project owner is group
+     */
+    private boolean isGroup;
+
+    /**
+     * To store the full name of the owner
+     */
+    private String ownerName = "";
+
     public HashSet<String> getNavigatorProjects() {
         return navigatorProjects;
     }
 
+    public boolean isGroup() {
+        return isGroup;
```

Comment:
```suggestion
        return gitlabOwner instanceOf GitLabGroup;
```

Reply:
Might not be able to do this as gitlabOwner is transient.